src/services/open-api-client.js

```diff
@@ -34,28 +35,23 @@ class OpenApiClient {
     const params = this.getParams(opts, operation);
 
     if (!opts.uri) {
-      opts.uri = this.getUri(opts);
+      opts.uri = this.getUri(opts.uri, opts);
     }
 
     // If the URI is relative, determine the host and prepend it.
     if (opts.uri.startsWith('/')) {
       if (!opts.host) {
         opts.host = path.server;
       }
-
-      if (opts.region) {
-        const parts = opts.host.split('.');
-
-        // From 'https://api.twilio.com/' to 'https://api.{region}.twilio.com/'
-        if (parts.length > 1 && parts[1] !== opts.region) {
-          parts.splice(1, 0, opts.region);
-          opts.host = parts.join('.');
-        }
-      }
-
+      opts.host = this.getHost(opts.host, opts);
```

Comment:
This is being done below.

Reply:
The one below doesn't take `opts.host` into account, it just parses from `opts.uri`

Reply:
Yeah, but the line after this one uses `opts.host`.

Reply:
🤦 I see it now